src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -107,12 +112,28 @@ public GHClient(IProgram program)
         }
     }
 
+    [PartCreationPolicy(CreationPolicy.Shared)]
+    public class VSGitExtPart
```

Comment:
I put this next to the export for `IGitHubClient`. Maybe we should collect all of the service exports together into a single file?

Reply:
This shouldn't be here (and GitHubClient shouldn't be either so ignore that example :P). Ideally it should be right next to the implementation, but since we don't want to have multiple exports for both teamfoundation assemblies, and we don't have a separate assembly yet for mef implementations, this should probably go into GitHub.Exports or into the Services folder here in GitHub.VisualStudio.

It should also follow the convention of the other mef dispatchers and be called VSGitExtDispatcher. For it to actually work as a MEF dispatcher, it needs to implement IVSGitExt and have a mef export for that interface.